schema.sql

```diff
@@ -343,7 +343,7 @@ CREATE TABLE IF NOT EXISTS `guildwar_kills` (
 CREATE TABLE IF NOT EXISTS `guild_invites` (
   `player_id` int(11) NOT NULL DEFAULT '0',
   `guild_id` int(11) NOT NULL DEFAULT '0',
-  `date` int(11) NOT NULL,
+  `date` int(11) NOT NULL DEFAULT '0',
```

Comment:
não é melhor deixar o default como CURRENT_TIMESTAMP? Se for zero, ele vai aparecer lá em 1969

Reply:
+1, melhor colocar o default pra a data atual

Reply:
Verdade

Reply:
@jlcvp me corrija se eu estiver errado,
entendo pouco de sql
mas o tipo dele ta INT(11) e acredito que não aceita CURRENT_TIMESTAMP.
Como deixaria o tipo?
Tipo date?
e se eu deixar DATE será que influenciará em alguma outra função?